libs/jquery.signalR-2.0.3.js

```diff
@@ -1,2693 +0,0 @@
-/* jquery.signalR.core.js */
```

Comment:
I checked in some old PRs that it was left because it was not available through NPM so I removed it now